src/screens/Home/ProgressTracker.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { TextBold } from 'assets/fonts/Fonts';
+import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
+import { Colors } from 'assets/Colors';
+import { View } from 'react-native';
+
+interface TrackerProps {
+  type: string;
+  status: string;
+  //   text?: string;
+}
+
+function setType(type: string) {
+  if (type === 'daca') {
+    return 'DACA Renewal';
+  }
+  if (type === 'general') {
+    return 'Citizenship';
+  }
+  return null;
+}
+
+function setStep(status: string) {
+  // change to what will be inputted in firestore
+  if (status === 'intakeSub') {
+    return 0;
+  }
+  if (status === 'uploadDoc') {
+    return 1;
+  }
+  if (status === 'underRev') {
+    return 2;
+  }
+  if (status === 'consultApr') {
+    return 3;
+  }
+  if (status === 'schedApt') {
+    return 4;
+  }
+  return null;
+}
+
+const ProgressTracker = (props: TrackerProps) => {
+  const { status } = props;
+  const { type } = props;
+  return (
+    <View style={{ height: 150 }}>
+      <TextBold>{setType(type)}</TextBold>
+      <ProgressSteps
+        activeStep={setStep(status)}
+        activeStepIconBorderColor={Colors.lightBlue}
+        completedProgressBarColor={Colors.brandBlue}
+        completedStepIconColor={Colors.lightBlue}
+      >
+        <ProgressStep label="Intake submitted" removeBtnRow />
```

Comment:
<img width="341" alt="Screen Shot 2021-11-18 at 9 13 33 PM" src="https://user-images.githubusercontent.com/63089231/142568774-c15becf1-aeb2-423c-a76f-dd45e9512a89.png">
THIS LOOKS SO COOL!!!! On mobile, the default text styling overlaps labels. Could you change the props of ProgressStep component to account for this? `nextBtnTextStyle` and `previousBtnTextStyle` might be relevant here

Reply:
https://www.npmjs.com/package/react-native-progress-steps